.github/workflows/release.yml

```diff
@@ -21,10 +24,28 @@ jobs:
       run: |
         mkdir ${OUTPUT_DIR}
         RELEASE_TAG="${GITHUB_REF##*/}"
-        TAR_FILE_NAME="stargz-snapshotter-${RELEASE_TAG}-linux-amd64.tar.gz"
+        TAR_FILE_NAME="stargz-snapshotter-${RELEASE_TAG}-linux-${{ matrix.arch }}.tar.gz"
```

Comment:
BuildKit and nerdctl uses `-linux-arm-v7.tgz` convention for ARM

Reply:
Fixed to use `arm-v7` identifier.